korangar/src/main.rs

```diff
@@ -1844,12 +1983,13 @@ fn main() {
 
                 drop(picker_render_pass);
                 drop(interface_render_pass);
-                drop(shadow_render_pass);
+                drop(directional_shadow_render_pass);
                 drop(geometry_render_pass);
                 drop(screen_render_pass);
                 let picker_command_buffer = picker_target.finish(picker_command_encoder);
                 let interface_command_buffer = interface_target.finish(interface_command_encoder);
-                let shadow_command_buffer = shadow_target.finish(shadow_command_encoder);
+                let directional_shadow_command_buffer = directional_shadow_target.finish(directional_shadow_command_encoder);
+                let point_shadow_command_buffer = point_shadow_target[0].finish(point_shadow_command_encoder); // FIX: Doesn't make any sense with the 0 here
```

Comment:
Maybe add, that it only works because we don't do anything target specific with the encoder (we only call "finish()" on it).